src/app/containers/ConsentBanner/Canonical/logic/index.js

```diff
@@ -0,0 +1,82 @@
+import Cookie from 'js-cookie';
+import setCookieOven from './setCookieOven';
+
+const PRIVACY_COOKIE = 'ckns_privacy';
+const EXPLICIT_COOKIE = 'ckns_explicit';
+const POLICY_COOKIE = 'ckns_policy';
+const COOKIE_EXPIRY = 365;
+const BANNER_APPROVED = '1';
+const POLICY_APPROVED = '111';
+const POLICY_DENIED = '000';
+
+const onClient = typeof window !== 'undefined';
+
+const setCookie = (name, value) =>
+  Cookie.set(name, value, { expires: COOKIE_EXPIRY });
+
+const setPolicyCookie = (value, logger) => {
+  setCookie(POLICY_COOKIE, value);
+  setCookieOven(POLICY_COOKIE, value, logger);
+};
+
+const seenPrivacyBanner = () => Cookie.get(PRIVACY_COOKIE) === BANNER_APPROVED;
```

Comment:
Great work with implementing the cookie requirements in a way that makes it clear to read. 💯 
This isn't the simplest of logic, but the consistency of naming and structuring of this file helps.